plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/VisitorTest.kt

```diff
@@ -101,13 +98,6 @@ class VisitorTest(
         @Parameterized.Parameters(name = "{0}/{1}")
         @JvmStatic
         fun parameters() = listOf(
-            arrayOf("androidxSqlite", "FrameworkSQLiteDatabase", AndroidXSQLiteDatabase(), null),
```

Comment:
`m`: ideally we'd also have a test for `FrameworkSQLiteOpenHelperFactory`, for example. We could pull the fixture from https://dl.google.com/android/maven2/androidx/sqlite/sqlite-framework/2.3.1/sqlite-framework-2.3.1.aar, extract a compiled `.class` from it and add it here + a test for this